doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,41 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This should underline "Requirements" also in the title.

Reply:
@CsatariGergely Thanks - Done